i18n/i18n.jsx

```diff
@@ -161,8 +161,12 @@ export function getLanguages() {
     if (!config.AvailableLocales) {
         return getAllLanguages();
     }
-
-    return config.AvailableLocales.split(',').filter((l) => languages[l]).map((l) => languages[l]);
+    return config.AvailableLocales.split(',').reduce((result, l) => {
```

Comment:
As an aside, the other usage of `getLanguages` "just worked" because arrays are objects too:
https://github.com/mattermost/mattermost-webapp/blob/082134496c96c36d92ad8e1ffceb41a65577b85d/components/user_settings/display/manage_languages.jsx#L68-L74

Great to make this explicit now! 👍